### PR TITLE
Lotame panorama id system updates

### PR DESCRIPTION
**Pull request by markaconrad**
## Type of change

- [x] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Store all cookies at the root domain
Don't touch our id cookie on a certain error code

- contact email of the adapter’s maintainer: mconrad@lotame.com
- [ ] official adapter submission

## Other information
This uses the new findRootDomain method now inside Lotame's User Id Module

### CODE REVIEW

**Comment by smenzer** [comment]
@markaconrad looks like there are circleci errors in your adapter...can you make sure to get those cleaned up, please?

File: modules/lotamePanoramaIdSystem.js [hunk 1]
```diff
@@ -211,10 +228,17 @@ export const lotamePanoramaIdSubmodule = {
           if (response) {
             try {
               let responseObj = JSON.parse(response);
-              saveLotameCache(KEY_EXPIRY, responseObj.expiry_ts);
+              const shouldUpdateProfileId = !(
+                utils.isArray(responseObj.errors) &&
+                responseObj.errors.indexOf(MISSING_CORE_CONSENT) !== -1
+              );
+
+              saveLotameCache(KEY_EXPIRY, responseObj.expiry_ts, responseObj.expiry_ts);
```
Comment (blingster7) [inline modules/lotamePanoramaIdSystem.js:236]: should this key-value ever get cleared from the cache?

File: test/spec/modules/lotamePanoramaIdSystem_spec.js [hunk 2]
```diff
@@ -446,4 +446,117 @@ describe('LotameId', function() {
...
+      sinon.assert.neverCalledWith(
+        setCookieStub,
+        '_cc_id',
+        '4ec137245858469eb94a4e248f238694'
+      );
+    });
+
+    it('should save the expiry', function () {
+      sinon.assert.calledWith(setLocalStorageStub, 'panoramaId_expiry', 10);
+
```
Comment (blingster7) [inline test/spec/modules/lotamePanoramaIdSystem_spec.js:488]: whitespace

File: modules/lotamePanoramaIdSystem.js (hunk 1 above)
Comment (markaconrad) [reply modules/lotamePanoramaIdSystem.js:236]: It is used to tell us when we should check the server again so removing is the same as not updating it and there is no practical benefit to explicitly clear this value from the cache

**Review by blingster7 — APPROVED** [review]
LGTM

**Comment by markaconrad** [comment]
Checking in to see when someone will have time for the 2nd review, thanks!

**Comment by smenzer** [comment]
@markaconrad / @blingster7 this isn't core code so it doesn't require a 2nd review. i'll go ahead and merge it now.

**Pull request merged**